pkg/utils/k8sutil.go

```diff
@@ -1058,3 +1073,19 @@ func addInitContainerAnnotation(dpm *v1beta1.Deployment) error {
 	}
 	return nil
 }
+
+//IsKubernetesResourceNotFoundError returns true if resource is not found
+func IsKubernetesResourceNotFoundError(err error) bool {
+	return k8sErrors.IsNotFound(err)
+}
+
+//CascadeDeleteOptions returns option for cascade deletion
+func CascadeDeleteOptions(gracePeriodSeconds int64) *metav1.DeleteOptions {
+	return &metav1.DeleteOptions{
+		GracePeriodSeconds: func(t int64) *int64 { return &t }(gracePeriodSeconds),
+		PropagationPolicy: func() *metav1.DeletionPropagation {
+			foreground := metav1.DeletePropagationForeground
+			return &foreground
```

Comment:
Oh right, I didn't realise it was a `const`.  In that case just put it into a variable:
```go
func CascadeDeleteOptions(...) {
   propForeground := metav1.DeletePropagationForeground
   return &metav1.DeleteOptions{
      PropagationPolicy: &propForeground,
      ...
   }
}
```